### PR TITLE
Disable irrelevant scene tab context menu items

### PR DESCRIPTION
**Pull request by MewPurPur**
Like it's done in the script list.

image

### CODE REVIEW

**Comment by KoBeWi** [comment]
Bug:
godot windows editor dev x86_64_CqvQlw9JEv

**Comment by MewPurPur** [comment]
Oh, so this makes an existing bug surface where the popup doesn't realize in time that by doing RMB, you changed the active scene. I have a mild memory of noticing this bug already being present for Undo Close Tab somehow. Either way, I should probably look into fixing this before the PR is acceptable

**Comment by KoBeWi** [comment]
I think you should use the clicked tab id instead of the current scene.

**Comment by MewPurPur** [comment]
True, doned.

**Review by KoBeWi — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -5619,7 +5620,13 @@ void EditorNode::_scene_tab_input(const Ref<InputEvent> &p_input) {
 					scene_tabs_context_menu->set_item_disabled(scene_tabs_context_menu->get_item_index(FILE_OPEN_PREV), true);
 				}
 				scene_tabs_context_menu->add_item(TTR("Close Other Tabs"), FILE_CLOSE_OTHERS);
+				if (editor_data.get_edited_scene_count() <= 1) {
+					scene_tabs_context_menu->set_item_disabled(file_menu->get_item_index(FILE_CLOSE_OTHERS), true);
+				}
 				scene_tabs_context_menu->add_item(TTR("Close Tabs to the Right"), FILE_CLOSE_RIGHT);
+				if (editor_data.get_edited_scene_count() == tab_id + 1) {
+					scene_tabs_context_menu->set_item_disabled(file_menu->get_item_index(FILE_CLOSE_RIGHT), true);
+				}
```
Comment (YuriSizov) [inline editor/editor_node.cpp:5629]: There is a typo here, right? It shouldn't be `file_menu`, it should be `scene_tabs_context_menu`.

Comment (YuriSizov) [reply editor/editor_node.cpp:5629]: I fixed it in https://github.com/godotengine/godot/pull/80490.

**Pull request merged**